src/editor/index.html

```diff
@@ -155,7 +156,6 @@
     <script src="libs/codemirror/addon/show-invisibles/show-invisibles.js"></script>
 
     <script src="init.js"></script>
```

Comment:
It won't really matter where it's moved to at this point... everything is asynchronous (and I mean everything).... I'll go ahead and move it though, lol.